index.html

```diff
@@ -845,11 +872,11 @@ <h3>Subject</h3>
       <section>
         <h3>Signatures / Proofs</h3>
         <ul>
-          <li>The document signature is available in the form of a known
-          signature suite.</li>
-          <li>Required signature properties are present. For example, for a
+          <li>The document proof is available in the form of a known
+          proof suite.</li>
+          <li>Required proof properties are present. For example, for a
           Linked Data Signature, <code>type</code>, <code>created</code>,
-          <code>creator></code>, and <code>signatureValue</code> are present.
+          <code>creator</code>, and <code>proofValue</code> are present.
```

Comment:
Again, wouldn't `signatureValue` be used here?

Reply:
Need to add text to say: "The ID is the ID of the holder, who may or may not be same as a subject in one of the embedded credentials".